src/client/components/Story/PostedFromEmbed.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Tooltip } from 'antd';
+import apps from '../../helpers/apps';
+import { jsonParse } from '../../helpers/formatter';
+import './PostedFromEmbed.less';
+
+const PostedFromEmbed = ({ post }) => (
+  <span>
```

Comment:
I think it would make sense to add a class name to a topmost element in this component (to respect http://getbem.com/introduction guidelines).